dangerfile.js

```diff
@@ -98,14 +101,8 @@ if (componentChangedOrCreated) {
   message(`
   ## Browser support
 
-  Please complete this list of browsers that you've checked this works in.
-
-  - [ ] IE11
-  - [ ] Edge
-  - [ ] Safari (iOS)
-  - [ ] Chrome (Android)
-  - [ ] Chrome (Desktop)
-  - [ ] Firefox (Desktop)
+  If this is a significant change, make sure you've tested it in multiple browsers,
```

Comment:
Maybe instead of this being from Danger we should keep the checklist and add it to the PR template instead to make it explicit as even small changes can have significant impact especially on IE - so we should make sure all changes are tested until IE 11 support is dropped

Reply:
I think we should adopt the same attitude we use elsewhere - which is that our tests should be sufficient to catch anything, so if CI passes everything should be fine

Reply:
I don't feel that is a good approach here as our CI doesn't do complex testing over different browsers - if it did then yes if CI is fine then we can merge but visually it could be broken on older browsers or non mainstream ones

Reply:
I think you're both right! 

Ideally our CI would do this but for now it's not feasible, so we have this message as a manual reminder. It would be cool to get to this as an end state though!

Reply:
With regards to having it in the PR template or Danger, I'd like to keep it in Danger because when I make PRs the first thing I do is remove the template, because I feel it's more for external contributors. 

I like it when Danger makes the comment because I get a notification to remind me, and it's something important, which is really the essence of what Danger is for.

Let's keep this but also add it to the PR template, there's no downside to having it in both places.

Reply:
Sounds good! Will we look at updating the wording for this as any visual change could have an affect across browsers not just significant

Reply:
Ah yeah I'll change that too

Reply:
Done — changed 'significant' to 'visual'